src/jwplayer/utils/useJwpAdsSetupComplete.ts

```diff
@@ -0,0 +1,23 @@
+import { useState, useEffect } from 'react';
+import { communicationService, ofType } from 'jwplayer/utils/communication';
+import { first } from 'rxjs/operators';
```

Comment:
I think you should rebase your branch from master as `rxjs` is no longer a thing here ;)